app/views/provider_interface/provider_relationship_permissions/_provider_permissions.html.erb

```diff
@@ -8,6 +8,14 @@
       see safeguarding information
     </li>
   <% end %>
+  <% if permissions.make_decisions %>
+    <li>
+      <svg class="app-icon govuk-!-margin-right-1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 200 200" focusable="false" fill="currentColor" aria-hidden="true">
+        <path d="M100 200a100 100 0 1 1 0-200 100 100 0 0 1 0 200zm-60-85l40 40 80-80-20-20-60 60-20-20-20 20z"></path>
```

Comment:
We should probably put this svg somewhere reusable — a helper?

Reply:
I'll tackle this when we refactor the onboarding flow.